File: 800/110A/110A.py
s = input()

count = 0
flag = True

for c in s:
    if c == '4' or c == '7':
        count += 1

count = str(count)

for c in count:
    if c != '4' and c != '7':
        flag = False
        break
    
print("YES" if flag == True else "NO")

File: 110A/110A.cpp
#include <iostream>
using namespace std;

int main(){
    string n;

    cin >> n;

    bool flag = true;

    int num = 0;

    for(char c : n){
        if(c == '4' || c == '7'){
            num++;
        }
    }

    if(num == 0) {
        cout << "NO" << endl;
        return 0;
    }

    while(num > 0){
        int check = num % 10;

        if(check != 4 && check != 7){
            flag = false;
            break;
        }

        num /= 10;
    }

    if(flag == true){
        cout << "YES" << endl;
    }
    else{
        cout << "NO" << endl;
    }
}